install.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+# Usage: ./install.sh  # klaam
+# Usage: ./install.sh --dev # klaam_dev
+
```

Comment:
Can you please let me know what this bash code is doing? is it setting the conda environment whether it is dev or not?

Reply:
This is just a simple script that you run to do the following:
- Build the environment using `conda`
- Install packages using `poetry`
- Install left-over packages that couldn't be installed using `poetry` using `pip`

`./install.sh` installs the actual environment of `klaam` that uses GPU, whereas `./install.sh --dev` installs the CPU version of the actual environment called `klaam_dev` to be able to test the build and run it on GitHub servers through GitHub Actions (CI).

Reply:
I see. Thanks.